user_manual/android_app.rst

```diff
@@ -33,6 +33,26 @@ When you download your ownCloud Android App from the Google Play store, it will
 
 It will either update automatically or give you a notification that an upgrade is available. If you are using an ownCloud Android app from a custom repository, e.g., your employer, then it will update in accordance with their policies.
 
+New Features Wizard
+-------------------
+
+.. versionadded:: 2.7.0
```

Comment:
Should this line and the `ref` ones be shown like the image below?

![screen shot 2018-05-04 at 11 59 38](https://user-images.githubusercontent.com/15261363/39622530-e368c4dc-4f92-11e8-8224-6f441eedc05e.png)

Reply:
No, they shouldn't. Not sure what's going on there.

Reply:
Oh, this is a screenshot from GitHub? If so, then that's to be expected, as it only has support for the reStructuredText format and not the sphinx-doc extensions. So, nothing to worry about.